task 2/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <opencv2\opencv.hpp>
```

Comment:
what's the difference between `\` and `/` ?

Reply:
my carelessness
it's wrong in linux 
but allowed in windows
N.E.V.E.R  USE  " \ "